### PR TITLE
Added Version and Travis Badges; Issue #46

### PR DESCRIPTION
**Pull request by aditya-iitd**
Codecov badge will be added once @ibrahimsharaf integrates codecov coverage.

### CODE REVIEW

**Comment by aditya-iitd** [comment]
Resolved some conflicts

**Comment by marco-c** [comment]
Can you create a clean pull request without so many unrelated commits? Follow the steps here when you are working on a new feature: https://github.com/marco-c/crashsimilarity/issues/9#issuecomment-287550565

File: README.md [hunk 1]
```diff
@@ -19,3 +19,17 @@ python3 -m unittest tests
 # Run a specific test
 python3 -m unittest tests/test_basic
 ```
+
+Badges:
```
Comment (marco-c) [inline README.md]: Can you put the badges at the top of the page? Right after the title `# crashsimilarity`.

File: README.md [hunk 2]
```diff
@@ -19,3 +19,17 @@ python3 -m unittest tests
...
+```sh
+# Version Badge
+.. image:: https://badge.fury.io/gh/marco-c%2Fcrashsimilarity.svg
```
Comment (marco-c) [inline README.md]: What is this doing? The badges should be added as images

Comment (marco-c) [inline README.md]: See the result: https://github.com/aditya-iitd/crashsimilarity/blob/8f23b20cc843b57476718bd60493f451d15ca227/README.md

File: README.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # crashsimilarity
 
+[![GitHub version](https://badge.fury.io/gh/marco-c%2Fcrashsimilarity.svg)](https://badge.fury.io/gh/marco-c%2Fcrashsimilarity)
+
```
Comment (marco-c) [inline README.md]: Can you remove this blank line? This way the badges will be on the same line in the resulting output: https://github.com/aditya-iitd/crashsimilarity/blob/5699c976cfde783d1db68e3dcdfea81c13d0c5ef/README.md.

Actually, let's remove the version badge for now. We will add it in the future if we decide to publish this package on PyPI.

**Pull request merged**